veneur/client.go

```diff
@@ -0,0 +1,77 @@
+package veneur
+
+import (
+	"time"
+
+	"github.com/segmentio/stats"
+	"github.com/segmentio/stats/datadog"
+)
+
+const (
+	TagVeneurGlobalOnly = "veneurglobalonly"
```

Comment:
How about dropping the `TagVeneur` prefix here? When using this package, it'll be just as expressive and less verbose to write `veneur.GlobalOnly` than `veneur.TagVeneurGlobalOnly`.